webview/src/plots/components/ribbon/RibbonBlock.tsx

```diff
@@ -48,14 +53,14 @@ export const RibbonBlock: React.FC<RibbonBlockProps> = ({
             </div>
           </>
         ) : (
-          <div className={styles.title}>
+          <div className={cx(styles.title, hasError && styles.errorIndicator)}>
```

Comment:
If we think making the titles red is too noisy, we could just update the icons:

<img width="736" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/5633ebf1-c361-425b-80f2-78dcb47881e2">